01-hello-world/src/main.ts

```diff
@@ -34,8 +33,10 @@ import {
   const deploy_txn = await Mina.transaction(deployerAccount, () => {
     AccountUpdate.fundNewAccount(deployerAccount);
     zkAppInstance.deploy({ zkappKey: zkAppPrivateKey });
-    zkAppInstance.sign(zkAppPrivateKey);
   });
+
+  await deploy_txn.prove();
```

Comment:
What happens if one calls this with proofsEnabled = false? Does it just take 0 seconds?

Reply:
This flag is useful when you want to run tests or a script quickly without having to wait for the proofs to be generated. Here is a little more [info](https://docs.minaprotocol.com/zkapps/how-to-test-a-zkapp#:~:text=Mina.LocalBlockchain()%20takes,setProofsEnabled(x%3A%20boolean)).  The refactor I have here runs similarly to how your code ran before.